blackbox_repository/tabulated_benchmark.py

```diff
@@ -143,12 +199,20 @@ def __init__(
             self,
             blackbox: Blackbox,
             elapsed_time_attr: str,
+            time_this_resource_attr: Optional[str] = None,
+            **kwargs,
     ):
         """
         Backend to run simulation from a user blackbox.
+
         :param blackbox: blackbox to be used for simulation, see `examples/launch_simulated_benchmark.py` for an example
-        on how to use.
+            on how to use.
         :param elapsed_time_attr:
+        :param time_this_resource_attr:
+        :param kwargs: Arguments for :class:`SimulatorBackend`
         """
-        super().__init__(elapsed_time_attr=elapsed_time_attr)
+        super().__init__(
+            elapsed_time_attr=elapsed_time_attr,
+            time_this_resource_attr=time_this_resource_attr,
+            **kwargs)
```

Comment:
We could pick an explicit naming and remove the comment.
```suggestion
            **simulatorbackend_kwargs)
```

Reply:
Good idea